web/bs.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+browser-sync start --server --files "*.*"
```

Comment:
What is this for?

Reply:
Secret!

Reply:
So, reveal this secret to me! :smiling_imp: